src/components/Tiles/TileGroup.js

```diff
@@ -14,7 +14,19 @@ const TileGroup = ({ title, titleUrl, description, tiles, compact, intl }) => {
         })}
       >
         <h4 className="coa-TileGroup__title">
-          {titleUrl ? <Link to={titleUrl}>{title}</Link> : title}
+          {titleUrl ? (
+            <Link
+              to={
+                titleUrl.substring(0, 4) === 'http'
+                  ? titleUrl
+                  : `/${intl.locale}${titleUrl}`
+              }
+            >
+              {title}
+            </Link>
+          ) : (
+            title
+          )}
```

Comment:
this is copypasta, and it'd be great to clean this up, but it's a rabbit hole so I made https://github.com/cityofaustin/techstack/issues/3083 instead